src/lightning/__init__.py

```diff
@@ -15,21 +16,28 @@
 
 from lightning.__about__ import *  # noqa: E402, F403
 from lightning.__version__ import version as __version__  # noqa: E402
-from lightning.fabric.fabric import Fabric  # noqa: E402
-from lightning.fabric.utilities.seed import seed_everything  # noqa: E402
-from lightning.pytorch.callbacks import Callback  # noqa: E402
-from lightning.pytorch.core import LightningDataModule, LightningModule  # noqa: E402
-from lightning.pytorch.trainer import Trainer  # noqa: E402
-
-__all__ = [
-    "Trainer",
-    "LightningDataModule",
-    "LightningModule",
-    "Callback",
-    "seed_everything",
-    "Fabric",
-    "__version__",
-]
+
+# This enables us to control imports in a more granular way for performance reasons
+lazy_import = bool(int(os.getenv("LIGHTNING_LAZY_IMPORTS", "0")))
```

Comment:
Is this meant to be set by the studio?

In that case, why not use the lightning_fabric, pytorch_lightning etc packages?